### PR TITLE
Den-12 Custom rule set update

### PR DESCRIPTION
**Pull request by darshandevexp**

### CODE REVIEW

File: edgecast/waf/custom_rule.go [hunk 1]
```diff
@@ -355,3 +355,24 @@ func (svc *WAFService) GetCustomRuleSet(accountNumber string, customRuleID strin
 
 	return parsedResponse, nil
 }
+
+//UpdateCustomRuleSet an access rule that identifies valid or malicious requests via whitelists, accesslists, and blacklists.
```
Comment (stevenpaz) [inline edgecast/waf/custom_rule.go:359]: Need to update this comment

File: example/waf/custom_rules/update/waf_custom_rule.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+								"value": "User-Agent"
+							}
+						],
+						"type": "REQUEST_HEADERS"
+					}
+				]
+			}
+		}
+	],
+	"name": "Deny Boys"
```
Comment (stevenpaz) [inline example/waf/custom_rules/update/waf_custom_rule.json:31]: Deny bots?

**Review by stevenpaz — CHANGES_REQUESTED** [review]

File: edgecast/waf/custom_rule.go (hunk 1 above)
Comment (darshandevexp) [reply edgecast/waf/custom_rule.go]: changed

File: example/waf/custom_rules/update/waf_custom_rule.json (hunk 2 above)
Comment (darshandevexp) [reply example/waf/custom_rules/update/waf_custom_rule.json]: corrrected

File: edgecast/waf/custom_rule.go (hunk 1 above)
Comment (stevenpaz) [reply edgecast/waf/custom_rule.go]: Hmm that description is for a custom rule set in general. I think just `// Updates a custom rule` should be enough

Comment (darshandevexp) [reply edgecast/waf/custom_rule.go]: i got this from help doc on ticket so more info should be good.

**Review by stevenpaz — APPROVED** [review]

**Pull request merged**